app/models/stats_admin.rb

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+class StatsAdmin
```

Comment:
Hmm, is this the right spot for this class?  For other routing constraint classes, we put them in ./sufia-models/lib/generators/sufia/models/templates/config/ and copy them into apps using generators.